Sources/Datadog/Core/DatadogCoreProtocol.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal var DefaultDatadogCore: DatadogCoreProtocol = NOOPDatadogCore()
```

Comment:
Is there any reasoning behind the capital `D` letter? `defaultDatadogCore` sounds more idiomatic for Swift 🙂.

Reply:
That's a mistake :)